### DISCUSSION TITLE
Forwarding deleted messages only

### DISCUSSION THREAD

**Discussion by kysterics** [aahnik/tgcf#157] [General]
Hope this is to be implemented...

Reference: https://github.com/ValeriyMaslenikov/tg-notify-deleted-messages

**Comment by aahnik** [comment]
That's a good idea. But Telegram does not notify always when a message was deleted. So while it may work, it will not be 100% reliable.

Do want deleted message from any chat to be forwarded to a specific chat? I don't know whether I will implement such or not. Because the current design of the project is not suitable for that.

**Reply by kysterics** [reply]
> Do want deleted message from any chat to be forwarded to a specific chat?

One specified is more than fine.
